README.md

```diff
@@ -156,11 +167,14 @@ val user: User = checkNotNull(
 
 ## findAll
 ```kotlin
-findAll(
-    sql: String, 
-    args: Map<String, Any?> = mapOf(), 
-    rowMapper: RowMapper<T>
-): List<T>
+@Suppress("UNUSED_PARAMETER")
```

Comment:
is the `Suppress` needed here and elsewhere?

Reply:
I think no. Cleaned it